Build/build.bat

```diff
@@ -3,3 +3,5 @@ copy /Y ..\Source\ExcelDna\Release64\ExcelDna64.xll ..\Distribution\
 copy /Y ..\Source\ExcelDna.Integration\bin\Release\ExcelDna.Integration.dll ..\Distribution\
 copy /Y ..\Source\ExcelDnaPack\bin\Release\ExcelDnaPack.exe ..\Distribution\
 copy /Y ..\Source\ExcelDnaPack\bin\Release\ExcelDnaPack.exe.config ..\Distribution\
+copy /Y ..\Source\ExcelDna.AddIn.Tasks\bin\Release\ExcelDna.AddIn.Tasks.dll ..\Package\ExcelDna.AddIn\tools\
+copy /Y ..\Source\ExcelDna.AddIn.Tasks\bin\Release\ExcelDna.AddIn.Tasks.pdb ..\Package\ExcelDna.AddIn\tools\
```

Comment:
Is there a particular reason to have the .pdb in the NuGet package?
(In general I'd prefer a smaller package).
Is there some reference to current recommendations on this?
Do we need to add the ExcelDna.Integration.pdb too?